idea/src/org/jetbrains/kotlin/idea/quickfix/AddExclExclCallFix.kt

```diff
@@ -50,42 +38,8 @@ import org.jetbrains.kotlin.util.OperatorNameConventions
 import org.jetbrains.kotlin.util.isValidOperator
 import org.jetbrains.kotlin.utils.addToStdlib.safeAs
 
-
-abstract class ExclExclCallFix(psiElement: PsiElement) : KotlinQuickFixAction<PsiElement>(psiElement) {
-    override fun getFamilyName(): String = text
-
-    override fun startInWriteAction(): Boolean = true
-}
-
-class RemoveExclExclCallFix(psiElement: PsiElement) : ExclExclCallFix(psiElement), CleanupFix, HighPriorityAction {
-    override fun getText(): String = KotlinBundle.message("fix.remove.non.null.assertion")
-
-    override fun isAvailable(project: Project, editor: Editor?, file: KtFile): Boolean =
```

Comment:
Just to make sure. This one is not overridden in the new `RemoveExclExclCallFix` on the right side. Is that intentional?

Reply:
Yes this is intentional. My understanding is that overriding `isAvailable` (or `isAvailableImpl`) is only really needed for intentions (as opposed to quick fixes where you already know the action is available since we created it for an element with diagnostic).